### PR TITLE
multifaucet WIP

### PR DESCRIPTION
**Pull request by rudi-cilibrasi**

### CODE REVIEW

File: src/action/faucet.mjs [hunk 1]
```diff
@@ -14,6 +19,27 @@ import {
   promTakeDrinkFromFaucet,
 } from '../prom/index.mjs';
 
+// not in promise core because uses dispatch and getState
+async function promPrivateTakeDrink({dispatch, getState}) {
```
Comment (Isabello) [inline src/action/faucet.mjs]: why? you can just send an array to the faucet

Comment (Isabello) [reply src/action/faucet.mjs]: i guess maybe its not implemented that way in iov core but it is in the faucet? 

https://github.com/iov-one/iov-core/blob/cca13e6/packages/iov-faucets/src/bovfaucet.ts#L19

Comment (rudi-cilibrasi) [reply src/action/faucet.mjs]: ``` result = await bovFaucet.credit(recipient);```

File: src/action/faucet.mjs [hunk 2]
```diff
@@ -14,27 +14,34 @@ import {
...
+// non optional ticker argument
+function performTakeDrinkFromFaucetForTickerForTelescope(ticker) {
+  if (!ticker) {
+    throw Error('Must provide ticker to performTakeDrinkFromFaucetForTickerForTelescope');
+  }
+  return performTakeDrinkFromFaucetForTelescope(ticker);
+}
+
+// optional ticker argument
+function performTakeDrinkFromFaucetForTelescope(ticker=undefined) {
```
Comment (Isabello) [inline src/action/faucet.mjs]: looks weird but i guess its ok

Comment (rudi-cilibrasi) [reply src/action/faucet.mjs]: only in operational terms. in logical terms it makes perfect sense because it "adds a constraint."

**Pull request merged**